site/content/en/docs/Tutorials/Matchmaker101/_index.md

```diff
@@ -42,7 +42,7 @@ REGISTRY=gcr.io/$(gcloud config list --format 'value(core.project)')
 
 ### Get the Tutorial template
 
-Make a local copy of the [Tutorials Folder](https://github.com/googleforgames/open-match/blob/{{< param release_branch >}}/tutorials/matchmaker101). Use `tutorials/matchmaker101` as a working copy for all the instructions in this tutorial.
+Make a local copy of the [Tutorials Folder](https://github.com/googleforgames/open-match/blob/{{< param release_branch >}}/tutorials/matchmaker101). Use `release-x.y` branch（e.g. release-1.3, release-1.4） of the Open Match Git repository. Use `tutorials/matchmaker101` of `release-x.y` branch as a working copy for all the instructions in this tutorial.
```

Comment:
```suggestion
Make a local copy of the [Tutorials Folder](https://github.com/googleforgames/open-match/blob/{{< param release_branch >}}/tutorials/matchmaker101). Checkout `release-x.y` branch（e.g. release-1.3, release-1.4） of the Open Match Git repository. Use `tutorials/matchmaker101` of `release-x.y` branch as a working copy for all the instructions in this tutorial.
```
Looks good, to be more precise suggesting `checkout` in place of `use`.

Reply:
Thank you for your review!
Changed to use `checkout`.